package.json

```diff
@@ -33,16 +33,16 @@
     "@rmwc/types": "^6.0.5",
     "@rmwc/typography": "^6.0.14",
     "@rooks/use-key": "^3.6.0",
-    "@types/codemirror": "0.0.106",
+    "@types/codemirror": "^5.60.1",
     "@types/jest": "^26.0.0",
     "@types/lodash.debounce": "^4.0.6",
     "@types/node": "^13.9.3",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
     "@types/react-redux": "^7.1.7",
     "@types/recompose": "^0.30.7",
-    "@uiw/react-codemirror": "^3.0.2",
```

Comment:
Our use case is a simple one and don't need a wrapper. It only makes logic harder to understand.